org.desmume.DeSmuME.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.desmume.DeSmuME</id>
+  <name>DeSmuME</name>
+  <summary>Nintendo DS emulator</summary>
+  <project_license>GPL-2.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>DeSmuME Team</developer_name>
+  <icon type="remote">https://linkmauve.fr/files/desmume-logo.svg</icon>
+
+  <url type="homepage">http://desmume.org</url>
+  <url type="bugtracker">https://github.com/TASVideos/desmume/issues</url>
+
+  <description>
+    <p>
+      DeSmuME is a Nintendo DS emulator.
+    </p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://linkmauve.fr/files/desmume-pokémon.png</image>
+      <caption>DeSmuME running Pokémon Heart Gold</caption>
+    </screenshot>
+  </screenshots>
+
+  <releases>
+    <release version="0.9.11" date="2015-04-15"/>
+    <release version="0.9.10" date="2013-11-28"/>
+    <release version="0.9.9" date="2013-04-30"/>
+    <release version="0.9.8" date="2012-04-09"/>
+    <release version="0.9.7" date="2011-02-01"/>
+    <release version="0.9.6" date="2010-05-19"/>
+    <release version="0.9.5" date="2009-11-27"/>
+    <release version="0.9.4" date="2009-07-08"/>
+    <!-- There is no 0.9.3, don’t look for it. -->
+    <release version="0.9.2" date="2009-04-12"/>
+    <release version="0.9.1" date="2009-02-13"/>
+    <release version="0.9" date="2009-01-04"/>
+    <release version="0.8" date="2008-04-22"/>
+    <release version="0.7.3" date="2007-08-12"/>
+    <release version="0.7.2" date="2007-07-10"/>
+    <release version="0.7.1" date="2007-06-13"/>
+    <release version="0.7.0" date="2007-05-11"/>
+    <release version="0.6.0" date="2007-02-04"/>
+    <release version="0.5.0" date="2006-12-23"/>
```

Comment:
```suggestion
```
There's no need to keep all the versions, especially if they don't a description